crates/devices/virtio-vsock/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# v0.2.0
+
+## Changes
+
+- Updated vm-memory to 0.10.0
+- Updated virtio-queue to 0.7.0
+- Upgrade Rust edition to 2021
```

Comment:
this should be mentioned as well: https://github.com/rust-vmm/vm-virtio/commit/6d316c008db5acb8a824719f16305cbfc4b488a9 (in an Added section).

Reply:
Mentioned the new `Eq` implementations in the appropriate changelog files